shell/initialize/App.vue

```diff
@@ -133,5 +78,18 @@ export default {
     },
   },
 
-  components: { NuxtLoading }
+  components: { NuxtLoading, NuxtError }
 };
+</script>
+<template>
+  <div id="__nuxt">
+    <NuxtLoading ref="loading" />
+    <div
+      id="__layout"
+      :key="showErrorPage"
+    >
+      <NuxtError v-if="showErrorPage" />
+      <router-view v-else />
+    </div>
+  </div>
+</template>
```

Comment:
This should be a straight translation from the functional render() definition to the template version. Tell me if you notice something that didn't translate 1 to 1.